File: 250903/배수의 개수/number-of-multipliers.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");
let cnt3 = 0, cnt5 = 0;

for (let i = 0; i < input.length; i++) {
    let num = Number(input[i]);
    if (num % 3 == 0) {
        cnt3 += 1;
    }
    if (num % 5 == 0) {
        cnt5 += 1;
    }
}
console.log(cnt3, cnt5);

File: 250903/배수의 개수/number-of-multipliers.py
arr = []
for i in range(10):
    num = int(input())
    arr.append(num)

a, b = 0, 0
for i in arr:
    if (i % 3 == 0):
        a += 1
    if (i % 5 == 0):
        b += 1
print(a, b)